tests/main/driver/test_analytic_init.py

```diff
@@ -7,8 +7,10 @@
 import pace.driver
 
 
+DIR = os.path.dirname(os.path.abspath(__file__))
+
 TESTED_CONFIGS: List[str] = [
-    "driver/examples/configs/analytic_test.yaml",
+    "../../../driver/examples/configs/analytic_test.yaml",
```

Comment:
Will we need to change this after the planned repo reorganization?  If so, add a TODO or some other tag here as a reminder.